src/data/sources.json

```diff
@@ -0,0 +1,349 @@
+{
```

Comment:
Should we generate this file at build time, you think? :thinking: At least then it would only ever be minimally out of date (as opposed to the situation where we forget to update this file manually :sweat_smile:)

Reply:
Agree! I'll add a script similar to the locale scripts.